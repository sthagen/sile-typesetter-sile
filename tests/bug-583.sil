\begin[papersize=a6]{document}
\use[module=packages.retrograde,target=v0.15.0]
\neverindent
\nofolios
\define[command=ah]{\discretionary[prebreak=-, replacement=’]}

(\hbox{Yasa\ah{}nın Tekrarı}) in hbox

(Yasa\ah{}nın Tekrarı) not in hbox

\end{document}
