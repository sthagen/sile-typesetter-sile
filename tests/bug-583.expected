Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	28.7889
Set font 	Gentium Plus;10;400;;normal;;;LTR;
T	11 w=3.1689	(()
Mx 	18.0508
T	60 68 86 68 w=18.9844	(Yasa)
Mx 	37.0352
T	183 w=2.2900	(’)
Mx 	39.3252
T	81 215 81 w=13.7451	(nın)
Mx 	55.7305
T	55 72 78 85 68 85 215 w=30.2246	(Tekrarı)
Mx 	85.9551
T	12 w=3.1689	())
Mx 	91.7842
T	76 81 w=8.2275	(in)
Mx 	102.6719
T	75 69 82 91 w=20.6543	(hbox)
Mx 	14.8819
My 	40.7889
T	11 w=3.1689	(()
Mx 	18.0508
T	60 68 86 68 w=18.9844	(Yasa)
Mx 	37.0352
T	183 w=2.2900	(’)
Mx 	39.3252
T	81 215 81 w=13.7451	(nın)
Mx 	55.7287
T	55 72 78 85 68 85 215 w=30.2246	(Tekrarı)
Mx 	85.9533
T	12 w=3.1689	())
Mx 	91.7805
T	81 82 87 w=13.9893	(not)
Mx 	108.4281
T	76 81 w=8.2275	(in)
Mx 	119.3139
T	75 69 82 91 w=20.6543	(hbox)
End page
Finish
